# In class discussion of chapters 1 - 4

## Genetic variation in natural populations (Ch. 3)

* Genetic diversity as subsection of biodiversity. Ecosystem → species → genes
* Genetic variation = building blocks for evolutionary and phenotypic change
   * Genetic difference between individuals within a pop
   * Genetic differences between pops within same species

### Processes that regulate genetic diversity
*Population connectivity (genetic exchange): maintain genetic diversity. Influences evolutionary processes, ability of pop to persist thru time
*Selection: drive genetic divergence. Causes allele frequency change over time, can lead to an overall loss of genetic diversity. Can lead to speciation (darwin’s finches)
*Tech influences on how we examine genetic diversity (table 3.1 and fig 4.1)
*From karyotypes, to PCR, polymorphisms, microsatellites, sequencing (has pushed out microsatellites)
*PCR (fig 3.1)
   * Perhaps most pervasive tech used in all of genetics
   * Nearly all sequencing tech depends on PCR at some level
   * Requires primer, denaturing DNA with heat, polymerase extends those 2 strands to become 2 double strands of DNA (doubles copies of DNA)
*Types of genetic variation
   * Chromosomes
      * Karyotype (fig 3.2 and 3.3)
      * One of first type of genetic variation examined
      * Have become increasingly relevant as we advance towards complete genome assemblies
   * Inversions (fig 3.6)
       *  A big part of adaptations in wild systems
       * Can create supergenes (clusters of genes inherited as one unit) that allows for rapid adaptation
   * Organelle DNA (box 3.2)
      * mtDNA
      * cpDNA
      * Easy to work with: usually inherited from one parent; multiple copies per cell (per nucleus); no recombination; little non-coding sequence and no introns
      * Genes from one parents makes drawing out phylogenies easier
      * Restriction enzyme analysis (RFLP) uses enzyme that has a known site where it cleaves DNA, can be used to infer different polymorphisms across genomes (fig 3.7). Used before sequencing
   * Nuclear DNA
        * Single copy nuclear loci (allozymes, then microsatellites)
        * Different proteins travel through gel depending on different properties, previously used to asses variation in natural pops (fig 3.10)
   * Sanger sequencing
        * Used diff form of PCR: dyed nucleotides create different lengths of DNA frags
     * NGS
        *  Illumina sequencing: picture analysis
        *  PacBio: reads fluorescent nucleotides over time
        * Nanopore: enzyme changes electric potential as diff nucleotides go through
   * SNP
      * One base pair in genome is variable
# Questions:
Formula for calculating rate of genetic divergence/diversity? Some kind of expected rate or expected outcome?
Yes and no; will learn more about estimating genetic diversity and trying to estimate selection from data



### Discussion

## #savethefrogs
# Ch. 3
   * Definition of domestication in context of agriculture– genetic definition?
   * Heterozygosity (differences in alleles (hetero = different, zygosity = zygote, in embryo)) in relation to fertility
      * Traditionally heterozygosity is good, but a reduction in fertility resulting from heterozygosity is bad. Heterozygosity in certain     chromosomal elements can result in loss of genetic information (fig 3.6, bottom 2 resulting gametes don’t have all the genetic info as top 2(missing F and A, respectively)) this could result in genetic abnormalities that reduces fertility
      * From this info we can assume that inversions don’t happen to often in the wild bc it results in a reduction of fitness
      * Usually the mechanism for why hybrids are infertile
      * Most things that affect infertility is determined by the impact on meiosis
    * Definition of race?
      * Philosophy of biology: no real difference between subpopulations/subspecies/etc. Even the term “species” is discussed and argued about
    * Quantitative threshold for inbreeding/outbreeding depression; what % of genetic variation is too little or too great?
      * Inbreeding depression: over time leads to decrease in heterozygosity and genetic diversity as a whole
        * too little genetic diversity results in recessive genes becoming more common and genetic disorders emerging where, in a diverse population, these recessive traits would die off
        * Cloning is not the same as inbreeding
      * Outbreeding depression: too much genetic variation decreases overall fitness of a population
        * local adaptation can facilitate outbreeding depression– pops adapted to one locality won’t perform as well in a different locality because genetically they’re not able to thrive in a space they’re not adapted to
        * can still happen when there’s low biodiversity in a pop
    * Why isn’t hermaphroditism more common/what makes it so common in plants and not in insects and other species?
      * Maintenance of eggs– high energy costs, need more space
    * Rarity of polyploidy in animals vs common in plants? How does it act as a mechanism of speciation?
      * probably related to life cycles– plants have different life stages where they have different numbers of chromosomes, where it’s very rare for animals
      * As a pathway to speciation?
      * Are odd ploidies (triploid as an example) always infertile? Almost, but not entirely
        * makes it very difficult for reproduction, but by cutting out the reproductive energy costs triploid species can put that energy towards other things like growing larger (oysters for example)
        * Many agricultural examples of manipulating ploidies to result in a larger/faster growing crops
    * How do allozymes describe the genetic structure of population and can be used to estimate genetic variation
    * Allozyme: an allelic variant of enzyme proteins; based on electrophoretic speed
    * Fig 3.10 shows variation of the same protein in diff individuals in the same pop– about 4 diff alleles for the same protein, defined by how far it’s traveled in the gel. You can also tell which are homozygous and which are heterozygous by how many alleles are present (2 would be homozygous, 1 is heterozygous)
      * This kind of imaging leaves genetics info out: redundancy in genetic code for amino acids. 1 base pair changes the amino acid at the nucleic level and could potentially change the function of the enzyme (nonsynonymous); synonymous have no changes of the amino acids at the nucleic level


## General questions
# Microsatellite:
    * Repeating patterns of alleles create either linear or kinked chains. The shape of these chains determine movement through a gel, and the distance they’ve moved indicates the genotype of an individual at multiple loci
# Epigenetics:
    * Genetic acclimation on a faster level. Histone proteins (what wraps DNA into chromatin) can affect phenotypes (not genetics). Methylation has different effects on DNA transcription that evokes a direct trans-generational change. The DNA itself isn’t changing, but some other process is changing to affect phenotypes of species
# Genome reduction options:
    * Fig 4.9: large part of genome that is repetitive DNA that doesn’t code for anything (introns), very hard to sequence. Exon contains most genetic information. “Complete genome” gets most of the actual genetic info, leaves out introns; scaffold genome gets most of the genetic info, but there are gaps. RADseq relies on enzymes to sample bits of the genome– get very little genetic info. Essentially sampling different parts of the genome
# eDNA
    * Extract DNA from environmental sample and amplifying the DNA of a species using a species specific primer
